tools/tutorial_videos/tutorial_videos/compose.py

```
@@ -0,0 +1,210 @@
+            "start_seconds": output_time(segments, float(step["start"])),
+        }
+        for step in timeline["steps"]
+    ]
+    # The narrator audibly "speaks into Lotti": mix the dictation clip at the
+    # (warped) moment it played into the virtual microphone.
+    dictation_clips = [
+        step["dictation"]["clip"]
+        for step in manifest["steps"]
+        if "dictation" in step
+    ]
+    for span, clip in zip(timeline.get("dictations", []), dictation_clips):
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)